lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2041,6 +2041,105 @@ namespace {
       fd->getAttrs().add(new (Impl.SwiftContext) UnsafeAttr(/*Implicit=*/true));
     }
 
+    void generateStaticFactoryForCXXForeignRef(
+        clang::ASTContext &clangCtx, ClassDecl *result,
+        clang::CXXRecordDecl *cxxRecordDecl, clang::Sema &clangSema) {
+
+      clang::IdentifierTable &idents = clangCtx.Idents;
```

Comment:
This can also moved down a bit. No need to generate a new identifier if we do not actually create a method (e.g., no suitable ctor is found.)